tools/docker/README.md

```diff
@@ -1,6 +1,9 @@
 # Using docker-compose for local development
+
 The docker-compose configuration present in this directory allows for a user to quickly setup all of chainlink's services to perform actions like integration tests, acceptance tests, and development across multiple services.
 
+Docker and this tool set are not required to develop on Chainlink, however they will greatly assist in setting up supporting environment.
```

Comment:
I haven't been able to run locally without docker. I get the error `open /run/secrets/keystore: no such file or directory`

Reply:
`cldev` does not support running outside of docker, we'd have to allow to switching between getting the keystore and password from `/run/secrets` to a configurable value. The problem with the old `cldev` is that it depends on pulling its own default env values, where you can see here: https://github.com/smartcontractkit/chainlink/blob/bcd9487c0f77e05e28301c3d178180c254fb66a4/tools/bin/cldev

So we would need a way of keeping a single source of truth for env configurations, else we risk making it harder to replicate environment setups.

Reply:
👍 I feel like we should include that here.

Reply:
re: cldev, why not use relative paths to the script?